pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[tool.poetry]
+name = "cycode"
+version = "0.2.3" # TODO(MarshalX): Dynamic from VCS
+description = "Perform secrets/iac scans for your sources using Cycode's engine"
+keywords=["secret-scan", "cycode", "devops", "token", "secret", "security", "cycode", "code"]
+authors = ["Cycode <support@cycode.com>"]
+license = "MIT"
+repository = "https://github.com/cycodehq-public/cycode-cli"
+readme = "README.md"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
```

Comment:
What is the actual status of our CLI? @artem-fedorov @MichalBor 

here is the list of available options:
[Development Status :: 1 - Planning](https://pypi.org/search/?c=Development+Status+%3A%3A+1+-+Planning) 
 [Development Status :: 2 - Pre-Alpha](https://pypi.org/search/?c=Development+Status+%3A%3A+2+-+Pre-Alpha) 
[Development Status :: 3 - Alpha](https://pypi.org/search/?c=Development+Status+%3A%3A+3+-+Alpha)   
[Development Status :: 4 - Beta](https://pypi.org/search/?c=Development+Status+%3A%3A+4+-+Beta)   
[Development Status :: 5 - Production/Stable](https://pypi.org/search/?c=Development+Status+%3A%3A+5+-+Production%2FStable)  
[Development Status :: 6 - Mature](https://pypi.org/search/?c=Development+Status+%3A%3A+6+-+Mature)  
[Development Status :: 7 - Inactive](https://pypi.org/search/?c=Development+Status+%3A%3A+7+-+Inactive)